cdap-docs/build.sh

```diff
@@ -30,6 +30,34 @@ ARG_1="${1}"
 ARG_2="${2}"
 ARG_3="${3}"
 
+function usage() {
+  cd ${PROJECT_PATH}
+  PROJECT_PATH=`pwd`
```

Comment:
Pretty sure this is wrong. If it isn't, you should use a different variable name if this can have different values at different times.